src/Microsoft.Health.Fhir.SqlServer/Features/ChangeFeed/SqlServerFhirResourceChangeDataStore.cs

```diff
@@ -79,23 +105,30 @@ public async Task<IReadOnlyCollection<ResourceChangeData>> GetRecordsAsync(long
                         }
                     }
 
-                    using (SqlCommand sqlCommand = new SqlCommand("dbo.FetchResourceChanges", sqlConnection))
+                    using (SqlCommand sqlCommand = sqlConnection.CreateCommand())
                     {
-                        sqlCommand.CommandType = CommandType.StoredProcedure;
-                        sqlCommand.Parameters.AddWithValue("@startId", SqlDbType.BigInt).Value = startId;
-                        sqlCommand.Parameters.AddWithValue("@pageSize", SqlDbType.SmallInt).Value = pageSize;
-                        using (SqlDataReader sqlDataReader = await sqlCommand.ExecuteReaderAsync(CommandBehavior.SingleResult, cancellationToken))
+                        PopulateFetchResourceChangesCommand(sqlCommand, startId, lastProcessedDateTime, pageSize);
+
+                        using (SqlDataReader sqlDataReader = await sqlCommand.ExecuteReaderAsync(CommandBehavior.SequentialAccess, cancellationToken))
```

Comment:
If it is the SignleResult, the query returns a single result set. SequentialAccess enables the DataReader to load data as a stream. In our case, there is no big difference since the query returns columns with large binary values. But, wanted to align with other use cases in the FHIR code base.